lib/segment/src/common/score_fusion.rs

```diff
@@ -0,0 +1,186 @@
+use std::collections::HashMap;
+use std::iter;
+
+use common::types::ScoreType;
+use itertools::{Itertools, MinMaxResult};
+use ordered_float::OrderedFloat;
+
+use crate::types::{Order, PointIdType, ScoredPoint};
+
+pub struct ScoreFusion {
+    /// Defines how to combine the scores of the same point in different lists
+    pub method: CombMethod,
+    /// Defines how to normalize the scores in each list
+    pub norm: Normalization,
+    /// Multipliers for each list of scores
+    pub weights: Vec<f32>,
+    /// Final ordering of the results
+    pub order: Order,
+}
+
+pub enum CombMethod {
```

Comment:
`CombMethod` is rather exotic, what about `Aggregation` instead?

Reply:
I didn't think too deep about it, but `Aggregation` makes sense. I was just following the naming used in the resources I read